src/app/core/entity/entity-actions/entity-actions.service.ts

```diff
@@ -59,22 +60,39 @@ export class EntityActionsService {
    *
    * This also triggers a toast message, enabling the user to undo the action.
    *
-   * @param entity The entity to remove
+   * @param entityParam The entity to remove
```

Comment:
```suggestion
   * @param entityParam The entity or entities to remove
```

Also, I think "entities" would be more clear than entityParam as a name

Reply:
but actually you use entityParam everywhere consistently, so let's leave that as is.